spec/dummy/db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.1].define(version: 2018_09_17_164000) do
+ActiveRecord::Schema[7.0].define(version: 2018_09_17_164000) do
```

Comment:
i did this pseudo manually...trying to run it for 7.0 caused bigints -> ints which we do not want

Reply:
What do you mean? I'd prefer if this file was generated and not a "pet" if that makes sense. Can you regenerate it and let's start from there?

Reply:
Previous commits **did** try to run migrations in tests, but it was running into issues with certain rails versions. Hence changing it to schema. I can go back to the run migrations way but I'm pretty sure that is going to cause other problems.

Reply:
Added a new commit that reverts back—I think it'll break for rails mainline...which is breaking other parts of this gem. I still argue it shouldn't even be tested.

Reply:
(I have to fix more stuff 😦 )